extensions/amp-pinterest/0.1/amp-pinterest.js

```diff
@@ -350,7 +350,7 @@ class AmpPinterest extends AMP.BaseElement {
         '?pin_ids=' + pinId +
         '&sub=www&base_scheme=https';
       return call(query).then(r => {
```

Comment:
@kentbrew this seems to work https://github.com/ampproject/amphtml/pull/1112/files can you try it out and see if it works for you here? if it still doesnt, let me know and lets go with your fix since getting the fix out is probably high priority.